### ISSUE TITLE
Disable some run commands

### ISSUE THREAD

**Issue by hackgence** [botgram/shell-bot#36]
is there any possibility of disabling specific run commands ?

for example, I don't want to run `/run rm xyz.txt`

how can we achieve that ?

**Comment by mildsunrise** [comment]
no, there's no functionality for whitelisting / blacklisting commands. you'll have to modify the code yourself for now
